config/flipt.schema.cue

```diff
@@ -216,7 +216,10 @@ import "strings"
 		}
 	}
 
-	#ui: enabled?: bool | *true
+	#ui: {
+		enabled?: 		bool | *true
+		default_theme?: string | "system"
```

Comment:
Quick suggestion as these are the only values this can be here. The `*` denotes the default when not supplied.
```suggestion
		default_theme?: "light" | "dark" | *"system"
```

Reply:
Ahhh, thanks for the explanation on `*` 👍